src/main/java/org/amoseman/budgetingbackend/resource/AccountResource.java

```diff
@@ -26,8 +27,8 @@ public AccountResource(AccountService<C> accountService) {
     @POST
     @Consumes(MediaType.APPLICATION_JSON)
     public Response createAccount(CreateAccount account) throws AccountAlreadyExistsException, UsernameExceedsMaxLengthException {
-        accountService.addAccount(account);
-        return Response.ok().build();
+        ResultType result = accountService.addAccount(account);
+        return Response.ok(result).build();
```

Comment:
Is this correct? Wouldn't you return a 400 if the user tried to use a password that didn't pass the criteria?